generators/client/templates/react/package.json

```diff
@@ -12,7 +12,7 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-hook-form": "7.28.0",
-    "react-jhipster": "0.19.0",
+    "react-jhipster": "jhipster/react-jhipster",
```

Comment:
I think you missed to change this? I don't think we should depend on master branch

Reply:
It needs https://github.com/jhipster/react-jhipster/commit/afba74cf026073d128d06423d32fbd6fe553aae3, otherwise translation will be missing.
We can switch to release at next release.

Can you do the release?

Reply:
I'll do a release tomorrow